src/jit/morph.cpp

```diff
@@ -2075,9 +2075,10 @@ GenTreePtr    Compiler::fgMakeTmpArgNode(unsigned tmpVarNum
                 // as that is how FEATURE_UNIX_AMD64_STRUCT_PASSING works.
             
                 // Pass by value in two registers
+                arg->gtFlags |= GTF_DONT_CSE;
```

Comment:
There shouldn't be any changes in this part of morph.cpp, so I evidently have more cleanup to do.